### PR TITLE
Added URL shortener blog

### PR DESCRIPTION
**Pull request by Haimantika**
## What does this PR do?

Adds URL shortener function template blog

## Test Plan

NA

## Related PRs and Issues

NA

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/blog/post/adding-url-shortener-function/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+For e.g https://dev.to/appwrite/introducing-appwrite-migrations-effortless-data-migration-from-your-platforms-5dhh to http://tinyurl.com/bd5usb3n 
+
+In this blog, we’ll learn to build a URL shortener using Appwrite Functions template. The template benefits from the built-in scalability, reliability, and security of Appwrite Functions.
+
+You can find the source code on our [templates GitHub repository](https://github.com/appwrite/templates/tree/main/node/url-shortener).
+
+# Setting up the Template
+To get started, you need to navigate to the functions page on the **[Appwrite](https://appwrite.io/cloud)** console. From there, we will select the **Templates** tab, search for and select the **URL Shortener** function template.
+
+/images/blog/adding-url-shortener/functions.png
```
Comment (TorstenDittmann) [inline src/routes/blog/post/adding-url-shortener-function/+page.markdoc:24]: the images here need to use the markdown image syntax

File: src/routes/blog/post/adding-url-shortener-function/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The function requires two variables:
+- APPWRITE_API_KEY
+- SHORT_BASE_URL
```
Comment (TorstenDittmann) [inline src/routes/blog/post/adding-url-shortener-function/+page.markdoc:28]: here we can use inline code

```suggestion
- `APPWRITE_API_KEY`
- `SHORT_BASE_URL`
```

File: static/images/blog/adding-url-shortener/Add-a-URL-Shortener-to-your-Application-with-Functions.png
Comment (TorstenDittmann) [inline static/images/blog/adding-url-shortener/Add-a-URL-Shortener-to-your-Application-with-Functions.png:1]: lets use lower case file names

File: static/images/blog/adding-url-shortener/URL-shortener.png
Comment (TorstenDittmann) [inline static/images/blog/adding-url-shortener/URL-shortener.png:1]: lets use lowercase filenames

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: src/routes/blog/post/adding-url-shortener-function/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+description: Learning how to add the URL shortener function template to your application.
+cover: /images/blog/adding-url-shortener/cover.png
+timeToRead: 5
+date: 2023-11-08
+author: haimantika-mitra
+category: functions
+featured: false
+---
+
+Appwrite Functions are user-defined functions that can start small and scale big, deploying automatically from source control. With the introduction of function templates, you can quickly add new integrations into your app without writing additional code or managing infrastructure. . Function templates are pre-built Appwrite Functions that can be integrated into your Appwrite project with just a few clicks.
```
Comment (LauraDuRy) [inline src/routes/blog/post/adding-url-shortener-function/+page.markdoc:13]: Mistake in punctuation

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**